developer_docs/preparing_a_pull_request.md

```diff
@@ -23,25 +23,34 @@ If you see an error, you'll need to start tracking the main p5.js repo as an "up
 
 Then ask git about the latest changes.
 
-    git fetch upstream 
+    git fetch upstream
 
 ### Just in case: make a copy of your changes in a new branch
-    git branch your-branch-name-backup 
+    git branch your-branch-name-backup
 
-### Apply changes from master branch, adds your changes *after* 
-    git rebase upstream/master 
+### Apply changes from master branch, adds your changes *after*
+    git rebase upstream/master
+
+### Switches back to master branch
+    git checkout master
+
+### Helps other contributors fully understand the changes that you made
+    git commit -m "Fixed documentation typos"   
+
+### Verifies what git will be committing  
+    git status       
 
 ## CONFLICTS
-You will probably have some conflicts! 
+You will probably have some conflicts!
```

Comment:
this section is actually incorrect and needs to be fixed. my bad for just copying it from the p5.js repository.

Reply:
@catarak What needs to be done here?